components/director/internal/domain/formation/fixtures_test.go

```diff
@@ -0,0 +1,33 @@
+package formation
+
+import "github.com/kyma-incubator/compass/components/director/internal/domain/formation/automock"
+
+func UnusedLabelService() func() *automock.LabelService {
+	return func() *automock.LabelService {
+		return &automock.LabelService{}
+	}
+}
```

Comment:
Unneccessary wrapping. You can directly return the service and use this function as a `func() *automock.LabelService`